### PR TITLE
Merges Malf AI and traitor AI into a single antag type, no longer has its own mode

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
And adds malf powers to traitor AIs. Malf is a dull mode that is usually 20-40 minutes of nothing followed by a few minutes of the same AI core assault by the handful of people with enough gear or access to do anything. Then the round ends. It's also a redundant mode considering AIs can and do go rogue and murder everyone in every other mode.

Pseduo-malf AI in traitor/DA rounds will open up possibilities like the AI working with badguys, badguys helping to stop the AI from nuking everything, stuff actually happening while the AI is hacking APCs, etc. It will also mean that AIs without cyborgs are not completely fucked.

Traitor AIs start with only 50 points to spend, but gain 10 more for every APC they hack.

For 180 points they can buy a nuke on a 400 second timer and blow the station up (if they live through the timer). This power triggers the usual malf sounds/runtime alert/ANGRY RED TILES.

TODO:

-Balance starting points/APC point gain/module cost. @GunHog please help with this you're the malf power guy

-Possibly tie cyborgs in with points (lose points upon borg activation, but allow you to sac them for points, make it cost points to hack them, etc). Cyborgs are a huge advantage for an AI but they're usually pretty helpless as an antagonist without them, so trading powers for borgs makes sense in my mind.

-Traitor/RD item to allow normal AIs to pick modules?

:cl: Kor
rscadd: Malfunctioning AIs have been merged with traitor AIs. They no longer appear in their own mode.
rscadd: Hacking APCs now gives you points to spend on your modules. Save up enough points and you can buy a doomsday device on a 450 second timer.
/:cl:

### CODE REVIEW

**Comment by KorPhaeron** [comment]
@Cheridan 

Can this be merged in the next couple days so I have time to address the balance issues that spring up before the freeze?

**Comment by KorPhaeron** [comment]
Running tally of traitor malf rounds:

1. Mekhi had the objective to nuke during traitor. Eaten by a singularity after he burned all his points on  machine overload to defend his core. Failure/death.

2. CKD had the objective to kill someone during double agent. Lost his borgs, but station evacced and left him alive. Victory/Alive

**Comment by KorPhaeron** [comment]
@tkdrg 

It all works now other than the announcement, which gives me the runtime

runtime error: Cannot read null.z
proc name: talk into (/obj/item/device/radio/talk_into)
  source file: radio.dm,419
  usr: null
  src: the station bounced radio (/obj/item/device/radio)
  call stack:
the station bounced radio (/obj/item/device/radio): talk into(the doomsday device (/obj/machinery/doomsday_device), "420 SECONDS UNTIL ACTIVATION!!...", null, /list (/list))
the doomsday device (/obj/machinery/doomsday_device): process(2)
Machines (/datum/subsystem/machines): fire()
Master (/datum/controller/master): process()

File: code/game/gamemodes/malfunction/Malf_Modules.dm [hunk 1]
```diff
@@ -534,3 +534,75 @@
...
+	density = 1
+	verb_exclaim = "blares"
+	var/timing = 1
+	var/timer = 500
+	var/obj/item/device/radio/radio
+
+
+/obj/machinery/doomsday_device/New()
+	..()
+	radio = new /obj/item/device/radio
```
Comment (tkdrg) [inline code/game/gamemodes/malfunction/Malf_Modules.dm]: this needs to be `radio = new /obj/item/device/radio(src)` i think

**Comment by KorPhaeron** [comment]
@tkdrg 

Ready

**Pull request merged**